docs/templates/plugin.rst.j2

```diff
@@ -106,7 +106,7 @@ Parameters
             <th width="100%">Comments</th>
         </tr>
         {% for key, value in options|dictsort recursive %}
-            <tr>
+            <tr id="parameter-@{ key }@">
```

Comment:
There's one downside: this will only work well for top-level options. Nested options might create collisions since their ID doesn't contain the keys of the options in which they are nested.